### DISCUSSION TITLE
I want to deploy it on my android phone.

### DISCUSSION THREAD

**Discussion by etjec4** [usememos/memos#2808] [Ideas]
how ?

**Comment by lincolnthalles** [comment]
There's not much use to it, as Memos is not a stand-alone application. But in case you have an old phone laying around that you can keep plugged in to use it as a sort-of a server, I published some instructions [here](https://github.com/lincolnthalles/memos-builds?tab=readme-ov-file#running-on-android).

Note that you'll have to use my builds or build Memos by yourself targeting Linux and your phone CPU architecture.
